.github/workflows/dependency_merge.yml

```diff
@@ -51,10 +37,10 @@ jobs:
 
       - name: Wait for other status checks to Pass
         id: waitforstatuschecks
-        uses: lewagon/wait-on-check-action@v1.3.3
+        uses: lewagon/wait-on-check-action@v1.3.1
         with:
           ref: ${{ github.head_ref }}
-          repo-token: ${{ secrets.CHERRYPICK_PAT }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is this change `s/CHERRYPICK_PAT/GITHUB_TOKEN` intended?

Reply:
making as uniform as https://github.com/SatelliteQE/robottelo/blob/6.14.z/.github/workflows/dependency_merge.yml please look at it